scripts/testing/generate_waterfall.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Please add a purpose and example usage comment to the top of these scripts for consistency.